src/app/clusters/scenes/SceneTableImpl.cpp

```diff
@@ -257,19 +257,19 @@ struct FabricSceneData : public PersistentData<kPersistentFabricBufferMax>
     /// @brief This Deserialize method is implemented only to allow compilation. It is not used throughout the code.
     /// @param reader TLV reader
     /// @return CHIP_NO_ERROR
-    CHIP_ERROR Deserialize(TLV::TLVReader & reader) override { return CHIP_NO_ERROR; }
+    CHIP_ERROR Deserialize(TLV::TLVReader & reader) override { return CHIP_ERROR_INCORRECT_STATE; }
 
     /// @brief This Deserialize method checks that the recovered scenes from the deserialization fit in the current max and if
-    /// there are too many scenes in nvm, it deletes them. The method sets the deleted_scene output parameter to true if scenes were
-    /// deleted so that the load function can know it needs to save the Fabric scene data to update the scene_count and the scene
-    /// map in stored memory.
+    /// there are too many scenes in nvm, it deletes them. The method sets the deleted_scenes output parameter to true if scenes
+    /// were deleted so that the load function can know it needs to save the Fabric scene data to update the scene_count and the
+    /// scene map in stored memory.
     /// @param reade [in] TLV reader, must be big enough to hold the scene map size
-    /// @param storage [in] Persistent Storage Delegate, required to delete scenes if the number of scene in storage is greater than
-    /// the maximum allowed
-    /// @param deleted_scenes [out] bool letting the caller (in this case the load method) know wether or not it was necessary to
-    /// delete scenes from the memory
+    /// @param storage [in] Persistent Storage Delegate, required to delete scenes if the number of scenes in storage is greater
+    /// than the maximum allowed
+    /// @param deleted_scenes_count [out] uint8_t letting the caller (in this case the load method) know how many scenes were
+    /// deleted so it can adjust the fabric and global scene count accordingly
```

Comment:
Probably worth documenting that this will be set to a meaningful and correct (as in, reflecting the number of scenes that were actually deleted) value even if this function returns failure.

Reply:
Documented.